values.yaml

```diff
@@ -513,6 +554,14 @@ connectInject:
   # method for Connect inject, set this to the name of your auth method.
   overrideAuthMethodName: ""
 
+  # aclInjectToken refers to a Kubernetes secret that you have created that contains
+  # an ACL token for your Consul cluster which allows the Connect injector the correct
+  # permissions. This is only needed if Consul namespaces and ACLs are enabled on the
```

Comment:
Can we list the exact permissions? Or link to where it's documented.

Reply:
Added info about the required permissions.